app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermDao.java

```diff
@@ -29,4 +29,7 @@
 
     @Query("SELECT COUNT(*) FROM sector_terms WHERE term >= :date")
     int countAllFutureSectorTerms(Date today);
+
```

Comment:
This cause compilation error when I switched to this branch. It should be :today instead of :date to fix the issue.
However I think it should be 
@Query("SELECT COUNT(*) FROM sector_terms WHERE term == :tomorrow")

Reply:
Yes, you're right. My fault I haven't dremoved this method. Look below, you'll find method you want :)